man/systemd.exec.xml

```diff
@@ -2916,6 +2916,29 @@ StandardInputData=V2XigLJyZSBubyBzdHJhbmdlcnMgdG8gbG92ZQpZb3Uga25vdyB0aGUgcnVsZX
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>LogFilterPatterns=</varname></term>
+
+        <listitem><para>Define a PCRE regular expression to filter log messages based on the MESSAGE= field of the
```

Comment:
i wonder if saying PCRE2 here is more appropriate. I think PCRE2 has a modified syntax, dunno, so it's a relevant details

Reply:
Just checked: PCRE2's pattern validation is more aggressive, so a valid PCRE pattern might not be valid with PCRE2. Updating the doc.